src/UserService.Data/Interfaces/IUserRepository.cs

```diff
@@ -36,5 +37,25 @@ public interface IUserRepository
         /// <param name="userEmail">Specified email of user.</param>
         /// <returns>User model.</returns>
         DbUser GetUserByEmail(string userEmail);
+
+        /// <summary>
+        /// Returns the user credentials with the specified user id from database.
+        /// </summary>
+        /// <param name="userId">Specified Id of user.</param>
+        /// <returns>User credentials model.</returns>
+        DbUserCredentials GetUserCredentialsById(Guid userId);
```

Comment:
Move credentials methods to new repository

Reply:
Splitted repository

Reply:
Fixed